src/styles/_header.scss

```diff
@@ -6,7 +6,7 @@
   padding: 0 50px;
   margin-bottom: 50px;
 
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  box-shadow: #{$header-shadow-color};
```

Comment:
don't need tipe #{} when connect variables in one file through import, use only $ (everywhere)